Source/Fuse.ImageTools/Tests/ImageTools.Test.uno

```diff
@@ -0,0 +1,48 @@
+using Uno;
+using Uno.Collections;
+using Uno.Testing;
+using Fuse;
+using FuseTest;
+
+namespace Fuse.ImageTools.Test
+{
+	public class ImageToolsTest : TestBase
+	{
+		private Object _image = null;
+
+		[Test]
+		[Ignore("Only supported on dotnet, Android or iOS", "native")]
```

Comment:
Possibly, a little confused 😅  This is supported on DotNet + Android + iOS, not just mobile. So it should be `"MOBILE || DOTNET"` I suppose? It's currently not supported on CMake nor MSVC - it's supported on every other target